engine/chainstate/chainstate.go

```diff
@@ -36,34 +35,31 @@ func (c *Client) SupportedBroadcastFormats() HexFormatFlag {
 	}
 }
 
-// Broadcast will attempt to broadcast a transaction using the given providers
-func (c *Client) Broadcast(ctx context.Context, id, txHex string, format HexFormatFlag, timeout time.Duration) (string, error) {
-	// Basic validation
-	if len(id) < 50 {
-		return "", ErrInvalidTransactionID
-	} else if len(txHex) <= 0 { // todo: validate the tx hex
-		return "", ErrInvalidTransactionHex
-	}
-
-	// Debug the id and hex
-	c.DebugLog("tx_id: " + id)
-	c.DebugLog("tx_hex: " + txHex)
-
-	// Broadcast or die
-	successCompleteCh := make(chan string)
-	errorCh := make(chan string)
-
-	go c.broadcast(ctx, id, txHex, format, timeout, successCompleteCh, errorCh)
+type BroadcastResult struct {
```

Comment:
For public structs, we should add a comment - or, maybe it is not required to be public 🤔

Reply:
I hate this- changed